src/js/components/plugins/CommonConnectivity.react.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Queries common inputs/outputs given list of bodyIds 
+*/
+
+"use strict"
+
+import React from 'react';
+import Button from 'material-ui/Button';
+import { FormControl, FormLabel, FormControlLabel } from 'material-ui/Form';
+import { Radio, RadioGroup } from 'material-ui/'
+import TextField from 'material-ui/TextField';
+import NeuronHelp from '../NeuronHelp.react';
+import PropTypes from 'prop-types';
```

Comment:
This is trivial, but the linter complains about the order of imports. Absolute imports should come before relative imports.